admin_guide/overcommit.adoc

```diff
@@ -127,6 +127,68 @@ exist.
 - *BestEffort* containers are treated with the lowest priority. Processes in
 these containers are first to be killed if the system runs out of memory.
 
+[[configuring-masters-for-overcommitment]]
+== Configuring Masters for Overcommitment
+
+Scheduling is based on resources requested, while quota and hard limits refer to
+resource limits, which can be set higher than requested resources. The difference
+between request and limit determines the level of overcommit; for instance, if a
+container is given a memory request of 1Gi and a memory limit of 2Gi, it is
+scheduled based on the 1Gi request being available on the node, but could use
+up to 2Gi; so it is 200% overcommitted.
+
+If OpenShift administrators would like to control the level of overcommit
+and manage container density on nodes, masters can be configured
+to override the ratio between request and limit set on developer
+containers. In conjunction with a link:./limits.html[per-project
+LimitRange] specifying limits and defaults, this adjusts the container
+limit and request to achieve the desired level of overcommit.
+
+This requires configuring the ClusterResourceOverride admission controller in the
+master-config.yaml as in the following example (reuse the existing configuration tree
+if it exists, or introduce absent elements as needed).
+
+====
+----
+kubernetesMasterConfig:
+  admissionConfig:
+    pluginConfig:
+      ClusterResourceOverride:   <1>
+        configuration:
+          apiVersion: v1
+          kind: ClusterResourceOverrideConfig
+          memoryRequestToLimitPercent: 25  <2>
+          cpuRequestToLimitPercent: 25     <3>
+          limitCPUToMemoryPercent: 200     <4>
+----
+<1> This is the plugin name; case matters and anything but an exact match for a plugin name is ignored.
+<2> (optional, 1-100) If a container memory limit has been specified or defaulted, the memory request is overridden to this percentage of the limit.
+<3> (optional, 1-100) If a container CPU limit has been specified or defaulted, the CPU request is overridden to this percentage of the limit.
+<4> (optional, positive integer) If a container memory limit has been specified or defaulted, the CPU limit is overridden to a percentage of the memory limit, with a 100 percentage scaling 1Gi of RAM to equal 1 CPU core. This is processed prior to overriding CPU request (if configured).
+====
+
+After changing the master config, a master restart is required.
+
+Note that these overrides have no effect if no limits have
+been set on containers. link:./limits.html[Create a LimitRange
+object] with default limits (per individual project, or in the
+link:./selfprovisioned_projects.html#template-for-new-projects[project
```

Comment:
New home at:

https://docs.openshift.org/latest/admin_guide/managing_projects.html#modifying-the-template-for-new-projects